src/model_signing/_serialization/file.py

```diff
@@ -56,6 +56,13 @@ def __init__(
         self._max_workers = max_workers
         self._allow_symlinks = allow_symlinks
 
+        # Precompute some private values only once by using a mock file hasher.
+        # None of the arguments used to build the hasher are used.
+        hasher = file_hasher_factory(pathlib.Path())
+        self._serialization = manifest._FileSerialization(
```

Comment:
optional: The base type being SerializationType provided some info, but when I first this line I wasn't sure what self._serialization was for the Serializer class. perhaps `_serialization_desc` or similar?